Taarafo.Core.Tests.Unit/Services/Foundations/PostImpressions/PostImpressionServiceTests.Logic.Add.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using Taarafo.Core.Models.PostImpressions;
+using Xunit;
+
+namespace Taarafo.Core.Tests.Unit.Services.Foundations.PostImpressions
+{
+    public partial class PostImpressionServiceTests
+    {
+        [Fact]
+        public async Task ShouldAddPostImpressionAsync()
+        {
+            //given
+            DateTimeOffset dateTime = GetRandomDateTimeOffset();
```

Comment:
call it `randomDateTime`

Reply:
I have renamed the variable in the commit named CODERUB: Changed Variable Name To randomDateTime